gulpfile.babel.js

```diff
@@ -19,6 +19,8 @@ import pkg from './package.json';
 
 const dirs = pkg['h5bp-configs'].directories;
 
+const jQueryVersion = pkg.devDependencies.jquery.replace("^","");
```

Comment:
Thanks. I hadn't had time to look at why the test was failing. I'm going to move back to explicit versions. It was always going to take some interaction to bump the versions anyway so there's no reason to go through back-flips on the test.